kong/plugins/standard-webhooks/handler.lua

```diff
@@ -0,0 +1,12 @@
+local plugin = require "kong.plugins.standard-webhooks.internal"
+
+local StandardWebhooks = {
+    VERSION = "1.0.0",
```

Comment:
```suggestion
    VERSION = require("kong.meta").version,
```
Good catch @outsinre !